apps/pipelines/nodes/nodes.py

```diff
@@ -426,8 +428,11 @@ def check_prompt_variables(self) -> Self:
             "tools": self.tools,
             "media": self.collection_id,
         }
+        # Only require collection_index_summaries variable if multiple indexes are selected
+        if len(self.collection_index_ids) > 1:
+            context["collection_index_summaries"] = self.collection_index_ids
```

Comment:
Nit: You're using `self.collection_index_ids` as a placeholder for the summaries? It's fine, but seems like a bug unless the reader knows what's going on. Would you mind adding a tiny comment that mentions that it's fine doing that here?

Reply:
I don't quite follow. This is added here to enforce that the `collection_index_summaries` prompt var is present when there are more than 1 collections added.